includes/import.form.inc

```diff
@@ -1192,7 +1238,7 @@ function islandora_multi_importer_form_submit($form, &$form_state) {
     // Actually in IslandoraMultiBatch all defaults to ingest
     // except for "update" which means updating existing objects.
     $parameters['action'] = isset($form_state['values']['action']) ? $form_state['values']['action'] : 'ingest';
-    dpm($parameters);
+    // dpm($parameters);
```

Comment:
these are the parameters i was talking about for save/restore.Except the actual uploaded files of course.